### ISSUE TITLE
Move pentacrest webcam experience to osc.uiowa.edu

### ISSUE THREAD

**Issue by mjoneill** [uiowa/uiowa#4825]
As a visitor to osc.uiowa.edu/webcams/pentacrest I see the pentacrest webcam currently at https://webcam.iowa.uiowa.edu/live/ so that ITS can retire the server/ownership of this piece of infrastructure

- [x] Understand if this is possible/what blockers exist if any
- [x] Understand if there are other cams/services on webcam.iowa.uiowa.edu
- [x] Embed feed on site?
- [x] #5437
- [x] Add a vulture redirect from webcam.iowa.uiowa.edu
- [x] Appropriate responsive sizing for video stream
- [ ] Decomission webcam.iowa.uiowa.edu

**Comment by mjoneill** [comment]
One concern I have is that this is more complex than embedding a feed and that the camera might need a server application running somewhere.  First step is likely a grooming step to try to understand this better.

**Comment by briand44** [comment]
@mjoneill were we waiting for something here? Was Nick T. going to get us a youTube link or something? I think we also wanted to consider how this affects Signage.

**Comment by briand44** [comment]
Update Signage to allow embed of YouTube stream in favor of current approach.

**Comment by briand44** [comment]
Send Communication ahead of time in case others are using that we are not aware of.

**Comment by mjoneill** [comment]
@mjoneill needs to followup with Hayley B to see if Public Safety has requirements

**Comment by mjoneill** [comment]
> @mjoneill needs to followup with Hayley B to see if Public Safety has requirements

I did so. Got the all clear from DSP.  They do not have any concerns or needs that intersect this work.

**Comment by briand44** [comment]
@mjoneill I added another checkbox to the IS to make sure we don't forget about Signage. There is currently a Signage pane that allows embedding a live view of the Pentacrest. This will break if we don't account for it somehow.

It may be as simple as deprecating that widget and allowing folks to use the existing video pane functionality. We would likely need to communicate that out prior to creating a redirect or decommissioning webcam.iowa.uiowa.edu. I haven't tested this yet to be sure it is a path forward but could easily if I had the YouTube link.

The other option would be to refactor the pane.

**Comment by mjoneill** [comment]
Working on this today.  

Note, Nick T from OSC shared these for embeds:

> I’m sorry for the barrage of emails.  We want to use this permanent embed code instead of the YouTube one.  If the stream is ever restarted, that embed will change.  The following embed code should be permanent:
> 
> `<iframe src="https://camstreamer.com/embed/WZoa7fiZquZiv6qZxpgA43yloExHdnwmmquO2Aco" style="border:none; overflow:hidden; width:560px; height:315px" allowfullscreen></iframe>`

The YouTube stream he originally shared shared THAT WE SHOULD NOT USE is: `https://youtu.be/lGOSajRs2c4`

**Comment by mjoneill** [comment]
I have created the page on the OSC website that contains the Pentacrest livecam feed.  

- Page exists at: https://osc.uiowa.edu/pentacrest-webcam
- I've asked Google to index this page via Search Console
- Page has a menu entry under Resources
- Meta Description added that optimizes for "Pentacrest Webcam".  I suspect it will take some time for search engines to catch up.
- I suspect there is something we could/should do to make the dimensions larger.  Can someone take a look at this next week (low priority).  Don't go down a rabbit hole if it's hacky and not apparent.
- Can we get a Vulture redirect from https://webcam.iowa.uiowa.edu/live/ in place (coordinate with Nick T from OSC).  We won't need this forever, but for now it is probably a good thing. Don't push on this if this intersects the SAN outage stack at all (people or tech).  We can wait if attention needs to be elsewhere.
- I've let Nick T know the page exists.

**Comment by briand44** [comment]
Based on the recommendation to use permanent embed code rather than a YouTube embed, it is starting to look like it makes more sense to refactor the Pentacrest Webcam signage pane.

This likely needs to happen prior to a Vulture redirect and definitely before retiring the server.

**Comment by bspeare** [comment]
You could use something like https://codepen.io/djlad/pen/grdOrj for the responsive iframe.   Add that CSS to the custom OSC theme in 703.

**Comment by briand44** [comment]
I pushed a branch with some css provided by Ben. Can someone take a look at this before I deploy? To take a look: 

- clone the OSC website locally on 703. 
- Update the Pentacrest Webcam page (node/986) to use

`<div class="responsive-iframe"><iframe allowfullscreen="" src="https://camstreamer.com/embed/zf6BUqMXKquweRwKYXtwqJ8c0VRIkXq0KdvMZs4d"></iframe></div>`


- checkout the branch pentacrest_styles

**Comment by joewhitsitt** [comment]
I can test this

**Comment by joewhitsitt** [comment]
👍 Fills out nicely:

<img width="569" alt="Screen Shot 2022-07-08 at 9 51 50 AM" src="https://user-images.githubusercontent.com/4663676/178016772-b4fe0dd2-f446-40e3-b9a0-a86f85cc048d.png">

**Comment by briand44** [comment]
Deployed, updated embed code to wrap in div and remove inline styles, cleared cache, purged that path. https://osc.uiowa.edu/pentacrest-webcam

**Comment by briand44** [comment]
DNS updated, TTL still low in case issues arise. Suggested waiting a week before anyone moves forward with retiring anything.

**Comment by briand44** [comment]
Communicated that TTL could be set back to default and that decommissioning could commence. Closing this as there isn't anything else on our end that I am aware of.
